### ISSUE TITLE
Cannot sign in through AAD sign in page on Android when screen reader is enabled.

### ISSUE THREAD

**Issue by chermali** [AzureAD/azure-activedirectory-library-for-android#147]
1. Enable TalkBack on Android device.  (This is the screen reader for Android.)
2. Open this link from Chrome on Android: https://webdir0d.tip.lync.com/PassiveAuth/PassiveAuth.aspx.
3. Enter credentials and password.
4. Click Submit.
5. The webpage will hang with the following error: Invalid url: java.lang.NumberFormatException: Invalid int: “null”.
